aiohttp_session/__init__.py

```diff
@@ -210,6 +210,7 @@ def save_cookie(self, response, cookie_data, *, max_age=None):
         params = dict(self._cookie_params)
         if max_age is not None:
             params['max_age'] = max_age
+            params['expires'] = time.strftime("%a, %d-%b-%Y %T GMT", time.gmtime(time.time() + max_age))
```

Comment:
Sorry for stupid question, does that mean
```
self._loop = loop if loop is not None else asyncio.get_event_loop()
``` 
? Because communication between `aiohttp` and `aiohttp_session` is very limited, don't see other way.